src/pages/_app.tsx

```diff
@@ -1,8 +1,13 @@
 import "../styles/globals.css";
 import type { AppProps } from "next/app";
+import NavbarWrapper from "../layouts/NavbarWrapper";
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+  return (
+    <NavbarWrapper>
```

Comment:
what's the benefit of using NavbarWrapper here rather than just Navbar directly? (If I'm understanding the MyApp is already a wrapper of sorts for the whole app ??)